### PR TITLE
Adding tags to mashups with +

### PR DESCRIPTION
**Pull request by MetalBall887**

### CODE REVIEW

File: tle/cogs/codeforces.py [hunk 1]
```diff
@@ -252,8 +256,13 @@ def pretty_time_delta(seconds):
...
                 (minutes, 'minute', 'minutes'),
             ]
+                        
+            timeprint = [(count,singular,plural) for count,singular,plural in timespec if count]
+            if not timeprint:
+                timeprint.append((seconds, 'second', 'seconds'))
+
             return ' '.join(f'{count} {singular if count == 1 else plural}'
-                            for count, singular, plural in timespec)
+                            for count, singular, plural in timeprint)
```
Comment (algmyr) [inline tle/cogs/codeforces.py]: Were these changes meant to go in this PR?

File: tle/cogs/codeforces.py [hunk 2]
```diff
@@ -152,6 +152,9 @@ def __init__(self, bot):
     async def mashup(self, ctx, *handles: str):
         """Create a mashup contest using problems within +-100 of average rating of handles provided.
         """
+        handles = [arg for arg in args if arg[0] != '+']
+        tags = [arg[1:] for arg in args if arg[0] == '+' and len (arg) > 1]
```
Comment (algmyr) [inline tle/cogs/codeforces.py]:
```suggestion
        tags = [arg[1:] for arg in args if arg[0] == '+' and len(arg) > 1]
```

File: tle/cogs/codeforces.py [hunk 3]
```diff
@@ -163,7 +166,8 @@ def __init__(self, bot):
                     if abs(prob.rating - rating) <= 100 and prob.name not in solved
                     and not any(cf_common.is_contest_writer(prob.contestId, handle) for handle in handles)
                     and not cf_common.is_nonstandard_contest(cf_common.cache2.contest_cache.get_contest(prob.contestId))]
-
+        if tags: problems = [prob for prob in problems if prob.tag_matches(tags)]
```
Comment (algmyr) [inline tle/cogs/codeforces.py]:
```suggestion
        if tags:
            problems = [prob for prob in problems if prob.tag_matches(tags)]
```

File: tle/cogs/codeforces.py [hunk 4]
```diff
@@ -152,6 +152,9 @@ def __init__(self, bot):
     async def mashup(self, ctx, *handles: str):
         """Create a mashup contest using problems within +-100 of average rating of handles provided.
```
Comment (algmyr) [inline tle/cogs/codeforces.py]: Update docstring and add usage notes.

File: tle/cogs/codeforces.py (hunk 1 above)
Comment (MetalBall887) [reply tle/cogs/codeforces.py]: My first "0 days 0 hours" changes were not commited in my fork for some reason, so those aren't changes, just my version doesn't have this line

**Pull request merged**